client/my-sites/plugins/plugin-list-header/index.jsx

```diff
@@ -142,8 +189,8 @@ export default React.createClass( {
 						href={ browserUrl }
 						onClick={ this.onBrowserLinkClick }
 						className="plugin-list-header__browser-button"
-						onMouseEnter={ () => this.setState( { addPluginTooltip: true } ) }
-						onMouseLeave={ () => this.setState( { addPluginTooltip: false } ) }
+						onMouseEnter={ this.tigglePluginTooltip.bind( this, true ) }
```

Comment:
We try to avoid fat arrow/bind for in jsx for perf reasons (avoids creating functions on each render). This won't make too much of a difference here, but we can avoid this with the following:

```js
onMouseEnter={ this.showPluginTooltip }
```
where
```javascript
showPluginTooltip() {
    this.setState( { addPluginTooltip: true } );
}
```